shiny/workflowPlot/server.R

```diff
@@ -246,6 +247,9 @@ server <- shinyServer(function(input, output, session) {
     # Not able to add icon over ggplotly
     # add_icon()
   })
+  
+  source("bm.R", local = TRUE)
```

Comment:
would be good to have a bit of doc here explaining what the source to bm.R is doing

Reply:
not fixed yet